draft-ietf-tls-extended-key-update.md

```diff
@@ -584,62 +507,346 @@ The exchange has the following steps:
 10. On receipt of the ACK message, the responder updates its send key and epoch
     value.
 
-Note that the procedure above aligns with the key update procedure defined in
-DTLS 1.3.
 
-{{dtls-key-update}} shows an example exchange illustrating a successful
-extended key update, which is reflected in the change of epoch values.
+## Legend used in Figures
+
+The following variables are used in the state machine diagrams below.
+
+- rx - current, accepted receive epoch.
+- tx - current transmit epoch used for tagging outgoing messages.
+- E - initial epoch value.
+- updating - true while a key-update handshake is in progress.
+- accepted - set to true after an accepted Resp; indicates the peer has
+  agreed to proceed with the update and that new key material can be derived.
+- old_rx - the previous receive epoch remembered during retention.
+- retain_old - when true, receiver accepts tags old_rx and rx.
+- tag=... - the TX-epoch value written on an outgoing message.
+- e==... - the tag carried on an incoming message (what the peer sent).
+- Req / Resp(true or false) / NKU / ACK / APP - protocol message types.
+- FINISHED / START/IDLE / WAIT_RESP / SENT_NKU / WAIT_R_NKU - diagram
+  states; FINISHED denotes the steady state after success or reject.
+
+
+## State Machine (Initiator)
```

Comment:
Any specific reason for the state machine not being depicted in the Appendix (for instance, see https://datatracker.ietf.org/doc/html/rfc8446#appendix-A.1) ?

Reply:
Add separate issue.